src/main/resources/messages_de.properties

```diff
@@ -28,6 +28,7 @@ statistics.matchesPlayed=Spiele
 statistics.wins=Siege
 statistics.losses=Niederlagen
 statistics.winRate=Siegesrate
+statistics.noRank=Du musst mindestens ein Spiel gespielt haben um deine Statistiken einzusehen
```

Comment:
That might be a bit too long to display it on the profile page what do you think about 'Not ranked yet' or 'Noch nicht platziert'?

Reply:
This currently replaces all of the statistics on the profile as I thought that it wouldn't make sense to show them to somebody who hasn't played any matches yet.